examples/gno.land/r/gnoland/valopers_proposal/proposal.gno

```diff
@@ -0,0 +1,56 @@
+package valopers_proposal
+
+import (
+	"std"
+
+	"gno.land/p/demo/dao"
+	"gno.land/p/demo/ufmt"
+	pVals "gno.land/p/sys/validators"
+	valopers "gno.land/r/gnoland/valopers"
+	"gno.land/r/gov/dao/bridge"
+	validators "gno.land/r/sys/validators/v2"
+)
+
+// MakeProposal creates a proposal to the GovDAO
+// for adding the given valoper to the validator set.
+func MakeProposal(address std.Address) {
+	var (
+		valoper     = valopers.GetByAddr(address)
+		votingPower = uint64(1)
+	)
+
+	// Determine the voting power
+	if !valoper.Active {
+		votingPower = uint64(0)
+	}
+
+	changesFn := func() []pVals.Validator {
+		return []pVals.Validator{
+			{
+				Address:     valoper.Address,
+				PubKey:      valoper.PubKey,
+				VotingPower: votingPower,
+			},
+		}
+	}
+
+	// Create the executor
+	executor := validators.NewPropExecutor(changesFn)
+
+	// Craft the proposal title
+	title := ufmt.Sprintf(
+		"Add valoper %s (Address: %s; PubKey: %s) to the valset",
```

Comment:
craft a link to the r/gnoland/valopers:addr

Reply:
added here: https://github.com/gnolang/gno/pull/3830/commits/4d9fccc1db077cfc847209df9d8e01701529e9f5#diff-561f31a1cba1a7bf843aee2807f5c1cac3b8be11502a1d8429557d4b14c5cbbcR58

Reply:
I also simplified the title of the proposal:
<img width="845" alt="image" src="https://github.com/user-attachments/assets/755cb96e-585e-47f2-91dd-d3cb4363f034" />

Reply:
Finally the title was not rendered properly with the link:
<img width="672" alt="Screenshot 2025-03-13 at 15 55 26" src="https://github.com/user-attachments/assets/aa0c87f9-c39f-4bc5-bb2b-5c9a0f127a75" />

So I removed the link from the title and put it at the beginning of the description:
<img width="789" alt="image" src="https://github.com/user-attachments/assets/641bac29-df6c-4142-9ed4-34c414d8267a" />